news/2023/2023-12-03-osu-indonesia-presents-ascaveth-invitational-tournament-2023.md

```diff
@@ -0,0 +1,52 @@
+
+LAN tournaments are no stranger to the Indonesian osu! community. Way back in 2012, before osu! conventions were a staple, the old guards of the community had hosted the "[osu! Rhythmical Adventure Competition](https://osu.ppy.sh/community/forums/topics/98417)" — a vast osu! tournament held in the middle of a shopping mall which was among the first (if not the first) LAN event of its kind.
+
+Now, more than a decade and [one successful small-scale iteration](/wiki/Tournaments/AIT/2022) later, osu! Indonesia is ready to present a spiritual successor to the event that rocked the world and [even peppy himself](http://web.facebook.com/osugame/photos/pb.100064751018669.-2207520000/10151075128588282/?type=3) eons ago — **Ascaveth Invitational Tournament 2023**!
+
+## Tournament
+
+Ascaveth Invitational Tournament 2023 is a double-elimination 1v1 osu! tournament hosted by ::{ flag=ID }:: [Ascaveth](https://osu.ppy.sh/users/3245206) and members of the Indonesian osu! community. The tournament features a lineup of eight top osu! players from all across Indonesia who have qualified through the [Last Chance Qualifiers (LCQ)](/wiki/Tournaments/AIT/2023_LCQ) event and other means as follows:
+
+- ::{ flag=ID }:: [Hakui Koyori](https://osu.ppy.sh/users/10717635)
+- ::{ flag=ID }:: [Skydiver](https://osu.ppy.sh/users/4750008)
+- ::{ flag=ID }:: [lifeline](https://osu.ppy.sh/users/11367222)
+- ::{ flag=ID }:: [rHO](https://osu.ppy.sh/users/1629553)
+- ::{ flag=ID }:: [NazunaAmemiya](https://osu.ppy.sh/users/12159899)
+- ::{ flag=ID }:: [Fuma](https://osu.ppy.sh/users/1501956)
+- ::{ flag=ID }:: [ThatNOOBGuy](https://osu.ppy.sh/users/11091594)
+- ::{ flag=ID }:: [DEETO](https://osu.ppy.sh/users/10069909)
+
+The tournament is scheduled to commence on **Saturday, December 9th** on **01:00 UTC** (08:00 local time). Unfortunately however, due to logistical and some other concerns, the tournament isn't open for public attendance. But fear not! As you might already expect, the entirety of the event will be streamed live over at **[the osuIndonesia Twitch channel](https://www.twitch.tv/osuIndonesia)** as well as **[lifeline's Twitch channel](https://www.twitch.tv/lifeline)**.
+
+Check out the trailer for the tournament below:
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/COMINGSOON" frameborder="0" allowfullscreen></iframe>
+</div>
+
+## Merchandise
+
+But wait, there's more!
+
+If you're in Indonesia, you're also able to support the tournament by purchasing exclusive merchandise ranging from keychains and stickers to t-shirts —  all starring the official mascot of the event, **[Stella Eirua](https://twitter.com/AscavethInv/status/1723545814449160621)**.
+
+Check out the following short trailer for the merchandise line below and, if this strikes you fance, you can place your order [here](https://docs.google.com/forms/d/e/1FAIpQLScmKN4kAXtiIluLchGfeodKpx37tcHd6OIe_6w1ZfLelDJWgA/viewform):
```

Comment:
```suggestion
Check out the following short trailer for the merchandise line below and, if this strikes you fancy, you can place your order [here](https://docs.google.com/forms/d/e/1FAIpQLScmKN4kAXtiIluLchGfeodKpx37tcHd6OIe_6w1ZfLelDJWgA/viewform):
```

Reply:
bruh what was i thinking when i made this typo lol